apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -138,21 +147,14 @@ class WindowsDownloads extends React.Component {
         )}
         {error && <FetchingLatestVersionError />}
         <div>
-          <h4>Instructions:</h4>
+          <h4>{i18n.instructions()}:</h4>
```

Comment:
[incorporate these strings into 1 for i18n] this one is a little weirder because it's just a colon, but in RTL languages the colon would be on the incorrect side, like "تعليمات:" (arabic)